src/Examples/Advanced/DataRecon/econ_recon_testing.ipynb

```diff
@@ -1292,32 +1153,11 @@
    "metadata": {},
    "outputs": [
     {
-     "name": "stdout",
+     "name": "stderr",
      "output_type": "stream",
      "text": [
-      "Plotting failed\n"
+      "Plotting data requires the 'seaborn' and 'PyPDF2' packages. Install the required packages before using the data_book() function. Plot terminated.\n"
```

Comment:
Is this something that should be addressed?

Reply:
@bpaul4 Yes, the is something that needs to be fixed soon, but not in this PR. My aim here was fix only the issues that are causing tests to fail on the PR in idaes-pse (plus fixing some of the more obvious deprecation as I went).